### ISSUE TITLE
Appointment Calendar - Suggestion

### ISSUE THREAD

**Issue by BG-RLTech** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#952]
We are starting to rely on customers booking through the widget on our website more heavily.

I think if you reschedule a ticket due time to a future date it should occupy that time slot on the appointment widget not allowing a customer to select that time in the widget.

Scenario 1: 
Sometimes we are unable to clear our repairs each day so we may carry over one or two repairs which we automatically allocate to the earliest appointment available the next day. We change the due time on the ticket to the next available slot yet this does not change the appointment calandar making it unavailable for customers to select on the widget online.

Scenario 2:
We get a repair in that requires us to order a part. We order the part necessary and it takes two weeks to arrive. We change the due date to the next available slot after the part is due. This time slot is still available on the online widget for customers to double book. This causes delays and angry customers because their devices run late.

To summarise. We have enough repairs in store to fill until lunch time tomorrow (5 repairs). A customer just messaged our page asking for a repair, we have directed them to our FB page and asked they book after 1pm because the morning is booked out however when they use the widget all the morning appointments will appear available.

**Comment by BG-RLTech** [comment]
If changing the due time which changes the position of the ticket on 'Dispatch (beta) does not correlate to the online widget then this is pointless if you only want one repair per time slot.

**Comment by BG-RLTech** [comment]
This is a feature that just needs to be tweeked to function great. We are currently considering moving to FB appointments and removing widget from website.
